src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1981,7 +1977,7 @@ because xbuild doesn't support framework reference assemblies.
 
 <Target Name="_CopyIntermediateAssemblies"
 	Inputs="@(ResolvedUserAssemblies);@(_AndroidResolvedSatellitePaths)"
-	Outputs="$(IntermediateOutputPath)_copyintermediate.stamp"
+	Outputs="$(_AndroidStampDirectory)_CopyIntermediateAssemblies.stamp"
```

Comment:
Is there a guideline for when a `.stamp` file should be explicitly removed in the `_CleanMonoAndroidIntermediateDir` target? This change removes `$(IntermediateOutputPath)`_copyintermediate.stamp`, which was removed in the `_CleanMonoAndroidIntermediateDir` target, but doesn't add `$(_AndroidStampDirectory)_CopyIntermediateAssemblies.stamp` to the `_CleanMonoAndroidIntermediateDir` target.

Should it?

Reply:
I documented about this here, but you probably missed that PR: https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/MSBuildBestPractices.md#stamp-files-1

The `$(_AndroidStampDirectory)` is at `$(IntermediateOutputPath)stamp\`, I put it in a sub-directory so we could delete it in one call and not have to worry about removing new stamp files when they are added: https://github.com/xamarin/xamarin-android/blob/f1359d99bddf59802e57479b5c4187f75426e3c9/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L3102